api.php

```diff
@@ -93,57 +113,59 @@
 	$value = $args['value'];
 	//set value check if 0
 	if ($value === '0') {
-		//set error message
-		$response->getBody()->write(json_encode(
-			['message' => 'zero is not a valid identifier']
-		));
-		return $response
-			->withHeader('Content-Type', 'application/json')
-			->withStatus(400);
+		//set error message -call bad request
+		return badRequestResponse(['message' => 'zero is not a valid identifier'], $response);
 	}
 	//create user object
 	$user = false;
 	// check if $value is a valid email
 	if (preg_match('/[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,3}/i', $value)) {
 		// find user by email
 		$user = User::findUserByEmail($value);
-	} else if (!preg_match('/\D/', $value)) { //check if value is integers only
+	} //check if value is integers only
+	else if (!preg_match('/\D/', $value)) {
 		//assign output of function to $user
 		$user = User::findUserById($value);
 	} else {
 		//set error message
-		$response->getBody()->write(
-			json_encode(
-				['message' => $value . ' is not a valid identifier, must be email or integer id']
-			)
-		);
-		return $response
-			->withHeader('Content-Type', 'application/json')
-			->withStatus(400);
+		return badRequestResponse(['message' => $value . ' is not a valid identifier, must be email or integer id'], $response);
 	}
+
 	// if we got nothing back from the user
 	if (empty($user)) {
 		// set not found message
-		$response->getBody()->write(json_encode([
-			'message' => 'user not found'
-		]));
-		// return 404
-		return $response
-			->withHeader('Content-Type', 'application/json')
-			->withStatus(404);
+		return notFoundResponse('user not found', $response);
 	}
 	//json encode the user data and write to the response body
-	$response->getBody()->write(json_encode($user));
-	//return the response w/ status code
-	return $response
-		->withHeader('Content-Type', 'application/json')
-		->withStatus(200);
+	return sendResponse($user, 200, $response);
 });
 
 // Create new user
 $app->post('/user/new', function (Request $request, Response $response, array $args) {
+
+	//set preg_match regex for name -will be used for first and last name
+	$nameRegEx = '/^[a-z ,.\'-]+$/i';
+
+	//regex pattern for phone number
+	$numberRegEx = '/\(?(\d{3})[\)\s-]*(\d{3})[\s\-]?(\d{4})/';
```

Comment:
Should we delete these variables since we set them higher up the page? @billythakidd04

Reply:
Nevermind I see you added it in back within the routes for scope.